### PR TITLE
Fix docs for running lli in LLVM 13 and add an end-to-end test for DynamicCallCounter

### LINKED ISSUE #62: dynamic-cc instrumented prints nothing
I'm using llvm 13.0.1
After being instrumented by dynamic-cc, running the program in lli prints nothing.
But when I run the same program in lli 15.0.0, it prints as expected.

Anybody experienced same issue?

### PR DESCRIPTION
**Pull request by YuuuuuuuuY**
Fix #62

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -500,7 +500,7 @@ to test **DynamicCallCounter**:
 ```bash
 export LLVM_DIR=<installation/dir/of/llvm/13>
 # Generate an LLVM file to analyze
-$LLVM_DIR/bin/clang -O1 -emit-llvm -c <source_dir>/inputs/input_for_cc.c -o input_for_cc.bc
+$LLVM_DIR/bin/clang -O0 -Xclang -disable-O0-optnone -emit-llvm -c <source_dir>/inputs/input_for_cc.c -o input_for_cc.bc
```
Comment (banach-space) [inline README.md:503]: The new RUN line that you added suggests that you can safely remove ` -O0 -Xclang -disable-O0-optnone` :)

Comment (YuuuuuuuuY) [reply README.md]: That's right. I just updated that and fixed ```opt``` with ```-enable-new-pm=0```.

**Comment by banach-space** [comment]
Thanks for the updates, that's much appreciated!

This is now ready to be merged :) One list thing - I'd like this to be one commit (to keep the history cleaner). Do you want me to squash it for you or do you prefer to squash it yourself and force-push to your feature branch? I'm fine either way!

**Comment by YuuuuuuuuY** [comment]
Please squash it. Thanks!

**Comment by banach-space** [comment]
Done, thanks for contributing :)

**Pull request merged**